spark/wordcount/README.md

```diff
@@ -0,0 +1,102 @@
+# Dataproc Bigtable WordCount Sample
+
+## Overview
+
+This sample demonstrates how to use  [Google Cloud Dataproc](https://cloud.google.com/dataproc), which
+provides a managed [Apache Spark](https://spark.apache.org/) environment with
+[Google Cloud Bigtable](https://cloud.google.com/bigtable/docs).
+
+This specific example provides the classic map reduce example of counting words
+in a text file.
+
+## Prerequisites
+
+1. A [Google Cloud project](https://console.cloud.google.com/) with billing enabled. Please
+be aware of [Bigtable](https://cloud.google.com/bigtable/pricing)
+and [Dataproc](https://cloud.google.com/dataproc/docs/resources/pricing) pricing.
+
+1. You have the [Google Cloud SDK](https://cloud.google.com/sdk/) installed.
+
+1. You have [Scala](https://www.scala-lang.org/) installed.
+
+1. You have basic familiarity with Spark and Scala. It may be helpful to
+install Spark locally.
+
+1. You have [Maven](https://maven.apache.org/) installed.
+
+## Create a Cloud Bigtable Instance
+
+If you don't already have a Cloud Bigtable instance, create one
+
+     gcloud bigtable instances create test-instance --cluster test-cluster --cluster-zone us-east1-b --cluster-num-nodes 3
+
+## Create a Cloud Dataproc Cluster
+
+Create a Cloud DataProc instance:
+
+    gcloud dataproc clusters create spark-cluster
+
+## Build the jar
+
+The Spark job is assembled into a fat jar with all its dependencies. To build, run
+
+    mvn assembly:assembly
+
+## Test your job locally (optional but recommended)
+
+This step requires a local Spark installation.
+
+While you will need a real Bigtable cluster, you can test the Spark job locally,
+if you have Spark insatlled. For testing, consider a Bigtable development
+cluster.
+
+    spark-submit --master local --class com.example.bigtable.spark.wordcount.WordCount \
+    target/cloud-bigtable-dataproc-spark-wordcount-0.1-jar-with-dependencies.jar \
+    your-project-id your-bigtable-instance-id wordcount-scratch \
+    src/test/resources/countme.txt
+
+The job will create the table specified (here, `wordcount`) if it doesn't already exist.
```

Comment:
The table above appears to be `wordcount-scratch`, not `wordcount`; is that correct?